tests/system/python/rpi/test_e2e_rpi_ephat.py

```diff
@@ -69,21 +69,23 @@ def get_statistics_map(self, foglamp_url):
 
     @pytest.fixture
     def start_south_north(self, reset_and_start_foglamp, add_south, south_branch, disable_schedule,
-                          remove_data_file, remove_directories, enable_schedule, foglamp_url,
-                          start_north_pi_server_c, pi_host, pi_port, pi_token, wait_time):
+                          remove_data_file, skip_verify_north_interface, remove_directories, enable_schedule,
+                          foglamp_url, start_north_pi_server_c, pi_host, pi_port, pi_token, wait_time):
         """ This fixture clones given south & filter plugin repo, and starts south and PI north C instance
 
         """
 
         add_south(SOUTH_PLUGIN, south_branch, foglamp_url, service_name=SVC_NAME)
 
-        start_north_pi_server_c(foglamp_url, pi_host, pi_port, pi_token, taskname=TASK_NAME, start_task=False)
+        if not skip_verify_north_interface:
```

Comment:
I think we should find some way to send data to north interface for these (another is modbus rtu) special tests /cc @ashwinscale